frontend/packages/ceph-storage-plugin/src/components/modals/block-pool-modal/create-block-pool-modal.tsx

```diff
@@ -69,8 +69,12 @@ export const CreateBlockPoolModal = withHandlePromise((props: CreateBlockPoolMod
         setIsSubmit(false);
         clearTimeout(timer);
         onPoolCreation(state.poolName);
-      } else if (newPoolLoaded && newPool?.status?.phase === POOL_STATE.FAILED) {
-        dispatch({ type: BlockPoolActionType.SET_POOL_STATUS, payload: POOL_PROGRESS.FAILED });
+      } else if (
+        newPoolLoaded &&
+        (newPool?.status?.phase === POOL_STATE.FAILED ||
+          newPool?.status?.phase === POOL_STATE.FAILURE)
+      ) {
+        dispatch({ type: BlockPoolActionType.SET_POOL_STATUS, payload: POOL_PROGRESS.NOTREADY });
```

Comment:
pool CR created but status is non-ready like failure

Reply:
this issue the fix for the timeout issue, the actual pool status is  failure but we are expecting failed